src/stylesheets/layouts/_header.scss

```diff
@@ -2,44 +2,47 @@
   background-color: $color-mint;
   display: flex;
   justify-content: center;
+  position: relative;
 
   &__wrapper {
-    margin: 50px;
-    // position: relative;
+    max-width: 50rem;
+    margin: 5rem 6rem;
+    position: relative;
   }
-  // &__primary {
-  //   display: flex;
-  //   justify-content: center;
-  //
-  //   &--top {
-  //     display: block;
-  //     margin-left: auto;
-  //     margin-right: auto;
-  //     text-align: center;
-  //   }
-  //
-  //   &--btm {
-  //     display: block;
-  //     width: 500px;
-  //   }
-  // }
-  &__text-box {
+
+  &__primary {
+    display: block;
```

Comment:
`<h1>`s should be `display: block` by default. Having some layout/width issues where you need this?